README-source.md

```diff
@@ -410,6 +410,8 @@ Adding a create method looks very similar.
 Every method you define on a `resource` Zapier converts to the appropriate Trigger, Create, or Search. Our examples
```

Comment:
Perhaps move the "Note" from below up to here.. maybe a sidebar:

# Note
>> Every method you define on a `resource` Zapier converts to the appropriate Trigger, Create, or Search. Our examples.

>> Moreover, Trigger, Create, Search, and Search or Create are automatically generated (in case you want to use them in a dynamic dropdown), like: `{resourceName}List`, `{resourceName}Create`, `{resourceName}Search`, and `{resourceName}SearchOrCreate`; in the examples above, `{resourceName}` would be `recipe`.